tools/make_release.py

```diff
@@ -222,20 +243,27 @@ def prepareManifestFile():
         # Prepare manifest.json file
         prepareManifestFile()
 
-        shutil.make_archive("catblock-opera", "zip", "catblock_opera")
+        # Generate a packed extension
+        if args.extension != None:
+            shutil.make_archive("catblock-opera", "zip", "catblock_opera")
+
+            # Rename to the Opera's compatible .nex file
+            if args.extension == "browser":
+                os.rename("catblock-opera.zip", "catblock-opera.nex")
+                shutil.move("catblock-opera.nex", "builds/catblock-opera.nex")
+            else:
+                shutil.move("catblock-opera.zip", "builds/catblock-opera.zip")
+
+        # When an unpacked development environment should not be created, remove the folder
+        if args.devenv == None:
+            shutil.rmtree("catblock_opera")
 
-        # Rename to the Opera compatible .nex file
-        if args.extension == "y":
-            os.rename("catblock-opera.zip", "catblock-opera.nex")
+        print("Done!")
 
-        if os.path.isfile("catblock-opera.zip"):
-            shutil.move("catblock-opera.zip", "builds/catblock-opera.zip")
-        else:
-            shutil.move("catblock-opera.nex", "builds/catblock-opera.nex")
+    elif args.browser == "safari":
 
-        shutil.rmtree("catblock_opera")
+        print("We are sorry, but building CatBlock for Safari hasn't been implemented yet.")
```

Comment:
If Safari is an unsupported browser, why not leave it for the `else` code block?